src/models/Album.js

```diff
@@ -0,0 +1,63 @@
+const mongoose = require('mongoose');
+
+const AlbumSchema = new mongoose.Schema({
+  artistName: {
+    type: String,
+    required:  [true, 'Please provide artist name'],
+    maxlength: [100, 'Artist name can not be more than 100 characters'],
+    trim: true,
+  },
+  albumName: {
+    type: String,
+    required:  [true, 'Please provide album name'],
+    maxlength: [100, 'Album name can not be more than 100 characters'],
+    trim: true,
+  },
+  price: {
+    type: Number,
+    required: [true, 'Please provide album price'],
+    min: 0,
+  },
+  image: {
+    type: String,
+    default: '/uploads/example.jpg',
+  },
+  releaseDate: {
+    type: Date,
+    required: false,
+  },
+  category: {
+    type: String,
+    required: [false, 'Please provide music category'],
+    enum: ['pop', 'rock', 'rap'],
```

Comment:
Since there can be a huge amount of genres and subgenres (rap-rock, heavy metal, bubblegum pop etc etc); I think it'll be difficult to cover all of them in an enum. My suggestion would be to leave this one as a plain string for now.

Reply:
@akosasante oh , sure,  I added an array here just to try on the frontend to add several options in a form, just to have most possible options there, does the 'plain string' means to leave it as enum: '',  like an empty string without specifying genres ('pop', 'rock', 'rap' etc.)?

Reply:
Yes correct remove just the `enum` line; since currently (I think) this will validate/only allow entries that are of one of those three genres/categories.

Reply:
Ah I reread your last comment and I think I missed or misundersood this part before: "just to try on the frontend to add several options in a form, just to have most possible options there". Sure that's a great idea and fits well with the music product! If that's something y'all want to try, you'll want to change the type like this, to allow an array of string entries: `type:[String]`